segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/UtilsWrapper.java

```diff
@@ -0,0 +1,287 @@
+
+import static java.util.concurrent.CompletableFuture.completedFuture;
+
+/**
+ * This class contains various utils methods useful for administration of LTS.
+ */
+@Slf4j
+@AllArgsConstructor
+@Data
+public class UtilsWrapper {
+    private static final SystemJournal.SystemJournalRecordBatch.SystemJournalRecordBatchSerializer BATCH_SERIALIZER = new SystemJournal.SystemJournalRecordBatch.SystemJournalRecordBatchSerializer();
+
+    @NonNull
+    private ChunkedSegmentStorage chunkedSegmentStorage;
+
+    private int bufferSize;
+
+    @NonNull
+    private Duration timeout;
+
+    /**
+     * Evict metadata cache.
+     *
+     * @return A CompletableFuture that, when completed, will indicate that the operation completed.
+     *         If the operation failed, it will be completed with the appropriate exception.
+     */
+    public CompletableFuture<Void> evictMetadataCache() {
+        return CompletableFuture.runAsync(() -> {
+            val metadataStore = (BaseMetadataStore) chunkedSegmentStorage.getMetadataStore();
+            metadataStore.evictAllEligibleEntriesFromBuffer();
+            metadataStore.evictFromCache();
+        }, chunkedSegmentStorage.getExecutor());
+    }
+
+    /**
+     * Evict {@link ReadIndexCache}.
+     *
+     * @return A CompletableFuture that, when completed, will indicate that the operation completed.
+     *         If the operation failed, it will be completed with the appropriate exception.
+     */
+    public CompletableFuture<Void> evictReadIndexCache() {
+        return CompletableFuture.runAsync(() -> {
+            chunkedSegmentStorage.getReadIndexCache().getSegmentsReadIndexCache().invalidateAll();
+            chunkedSegmentStorage.getReadIndexCache().getIndexEntryCache().invalidateAll();
+        }, chunkedSegmentStorage.getExecutor());
+    }
+
+    /**
+     * Evict {@link ReadIndexCache} for given segment.
+     *
+     * @param segmentName Name of the segment.
+     * @return A CompletableFuture that, when completed, will indicate that the operation completed.
+     *         If the operation failed, it will be completed with the appropriate exception.
+     */
+    public CompletableFuture<Void> evictReadIndexCacheForSegment(String segmentName) {
+        Preconditions.checkNotNull(segmentName, "segmentName");
+        return CompletableFuture.runAsync(() -> chunkedSegmentStorage.getReadIndexCache().remove(segmentName), chunkedSegmentStorage.getExecutor());
+    }
+
+    /**
+     * Copy the contents of given segment to provided {@link OutputStream}.
+     *
+     * @param segmentName Name of the segment.
+     * @param outputStream Instance of {@link OutputStream} to copy to.
+     * @return A CompletableFuture that, when completed, will indicate that the operation completed.
+     *         If the operation failed, it will be completed with the appropriate exception.
+     */
+    public CompletableFuture<Void> copyFromSegment(String segmentName, OutputStream outputStream) {
```

Comment:
Does this iterates through all the chunks of a Segment? If so, where is the iteration over the chunks?